src/components/CountrySelect.vue

```diff
@@ -67,3 +74,4 @@ export default {
   }
 }
 </script>
+:inputStyle='mediumStyles'
```

Comment:
Pretty sure this would break the build. Is this a typo?

Reply:
Sorry... fixing, fixing!!

Reply:
Don't know how it built and ran properly with that, but now it's been removed